src/handle_action.cpp

```diff
@@ -419,8 +420,28 @@ static void pldrive( int x, int y )
             return;
         }
     }
-
-    veh->pldrive( point( x, y ) );
+    if( !remote && z != 0 && !u.has_trait( trait_PROF_HELI_PILOT ) ) {
+        u.add_msg_if_player( m_info, _( "You have no idea how to make the vehicle fly." ) );
+        return;
+    }
+    if( !g->m.has_zlevels() ) {
```

Comment:
I think this will block all vehicles from being drive-able, I understand helicopters without zlevels don't make sense but if someone plays with zlevels off i think they won't be able to drive anything when the patch gets merged

Reply:
good catch, narrow focus while testing only helicopters :D
added check for pldrive z parameter not being zero before early return.